recipe.yml

```diff
@@ -5,12 +5,36 @@ vibversion: 1.0.0
 stages:
 - id: build
   base: scratch
-  singlelayer: true
-  labels:
-    maintainer: Vanilla OS Contributors
```

Comment:
Why was this removed?

Reply:
Only the second stage will be uploaded, so it was moved there.